### DISCUSSION TITLE
CSS FlexBox

### DISCUSSION THREAD

**Discussion by mohbusaidy** [neuefische/questions#28] [Web] [answered]
Hi liebe Fischchen, 
ich habe probleme mit meiner Positionierung. Mein Ziel ist es, dass es meine Karten nebeneinander anzeigt. Und ich bekomme das leider nicht hin und die Karte stellen sich untereinander auf. 

Bisher: habe ich mit anderen Studenten zsm getan, leider nicht erfolgreich geworden. 

Außerdem habe ich im protokoll zu dem thema CSS flex box und mit mir die relevanten codepen's angeguckt.  
<img width="300" alt="Bildschirmfoto 2022-07-28 um 11 22 01" src="https://user-images.githubusercontent.com/104504053/181471317-4f30ebd7-dfa0-43d6-9b05-09d8c4c5d02d.png">

**Comment by julialoeschel** [comment, marked as answer]
Hallo Mo. 
Ich habe dir mal eine Sanbox gebaut - schau mal ob du deinen Code soweit abspecken kannst, das du nur noch da hast was du wirklich benötigst. 
Dann kannst du mit der Sandbox bauen, was du genau brauchst. 
https://codesandbox.io/s/hardcore-poitras-6iq6yv?file=/styles.css

**Reply by mohbusaidy** [reply]
Vielen dank! Probier ich sofort aus.

**Reply by mohbusaidy** [reply]
Ich hab deine vorschläge implementiert, aber das hat leider auch nicht geklappt :(

**Reply by anresa** [reply]
@mohbusaidy  Das ist das Gleiche, dass wir vorher schonmal ausprobiert haben. @julialoeschel Eigenartig ist auch, dass sich zwischendrin alle Cards komplett übereinander legen und gar keine Trennung/keinen Abstand mehr haben. Vielleicht hilft das irgendwie weiter? :)

**Comment by pixelass** [comment]
@mohbusaidy Kannst du das Problem in einer Codesandbox isolieren (Den wichtigen Teil dort übertragen und dann in deiner Frage verlinken)
Das vereinfacht es das Problem zu beurteilen und einen Lösungsansatz vorzuschlagen.

Siehe: https://github.com/neuefische/questions/wiki/How-to-ask-good-questions#example-3

**Reply by mohbusaidy** [reply]
In der code sandbox habe ich das gleiche problem! Ich finde mein denk fehler nicht 



https://codesandbox.io/s/nice-bouman-8057cu

**Reply by JessicaLoers** [reply]
**Hi @mohbusaidy** ✌️ in der `App.js` der Codesandbox fehlen mehrere Referenzen `Header.js ,  Main.js ,  Favorites.js , data.js`.  Diese Fehler müssten bitte erst behoben werden, dann kann dein CSS Problem reproduziert werden und hoffentlich auch gelöst.

**Reply by mohbusaidy** [reply]
Ich hab es endlich raus meine frage ist jetzt auch durch.

**Reply by mbosselmann** [reply]
Super! 💪 Magst du noch markieren, welcher Lösungsvorschlag dir bei der Lösung geholfen hat? Und wenn du dein Problem doch anders gelöst hast, dann kannst du diese Lösung auch gerne als Antwort posten und durch das Klicken auf "Mark as answer" als Antwort markieren.
